### ISSUE TITLE
Missing service from Service Map

### ISSUE THREAD

**Issue by makeavish** [SigNoz/signoz#1667]
## Bug description

After instrumenting a single nodeJs app, it's not being shown in service map. App is shown on front page and instrumented successfully.

## Expected behavior

App should be shown in service map.

## How to reproduce

1. Instrument a single nodeJs app
2. Check SigNoz service map page

## Version information 
* **Signoz version**: v0.11.2
* **Your CPU Architecture**(ARM/Intel): Intel

**Comment by makeavish** [comment]
By looking at [materialized views](https://github.com/SigNoz/signoz-otel-collector/blob/main/exporter/clickhousetracesexporter/migrations/000016_dependency_graph_minutes.up.sql) responsible for populating services for service map it appears that we don't show single services on Service Map page.
@srikanthccv can you please confirm this?

**Comment by srikanthccv** [comment]
That's correct. The individual service-wise numbers are shown in the landing page table. The service graph shows the RED metrics on edges, giving service interaction such as which downstream service is creating the most traffic, what error rate is observed b/w service calls, etc.

**Comment by pranay01** [comment]
@makeavish @makeavish So, if a user has a single service instrumented then it will not be shown in the service map?

**Comment by srikanthccv** [comment]
Yes, that's the current behavior.

**Comment by makeavish** [comment]
@pranay01 Should we show individual services in Service Map UI? 
Will it be relevant for the user who is instrumenting a single service for POC? One of the user thought it’s a bug.

**Comment by pranay01** [comment]
That's a good point @makeavish Many people start with 1 service to test SigNoz.

How can we handle it?

Should we show an explicit message -  if there is only one service? Like 

`Service map shows how different services interact with each other. Please instrument more than 1 service to show the relations in service map`

**Comment by makeavish** [comment]
Yes explicit message might be better option right now.
